qchem/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 <h3>New features since last release</h3>
 
+* The function ``particle_number`` has been added to the
+  ``obs`` to build the particle number operator using the function 
+  ``observable`` as implemented in the same module.
```

Comment:
Since this is a markdown file, only need single backticks 🙂 
```suggestion
* The function `particle_number` has been added to the
  `obs` to build the particle number operator using the function 
  `observable` as implemented in the same module.
```

Reply:
However, I might re-word this --- the changelog doesn't need to specify the implementation details, just highlight and describe the new feature. For example:

```markdown
Adds the function `qml.qchem.particle_number`, which computes the particle number operator
of a given molecule.
```